wg-outreach/README.md

```diff
@@ -15,6 +15,16 @@ Grows the Electron community
 | <img src="https://github.com/erickzhao.png" width=100 alt="@erickzhao">  | Erick Zhao [@erickzhao](https://github.com/erickzhao) | Member | ET (Montreal) |
 | <img src="https://github.com/VerteDinde.png" width=100 alt="@VerteDinde">  | Keeley Hammond [@VerteDinde](https://github.com/VerteDinde) | Member | PT (Portland) |
 
+## Current Objective and Key Results
+**Objective:**
+
+Electron is easy for new people to approach and make meaningful contributions.
+
+**Key Results:**
```

Comment:
For consistency with other OKRs as well as a bit more elaboration:
```
* Reduce time needed to set up developer environment
* Increase breadth of contributors across organizations
* Increase the number of regular contributors (rolling 90-day active)
```

Reply:
:+1: 

The addition of "across organizations" here isn't in the original OKR notes but IMO it's consistent with the governance session where there was concern that any one company could have de facto control over the project